src/components/Popover/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as MPopoverLayer } from './src/PopoverLayer.vue';
+export { default as MPopover } from './src/Popover.vue';
+export { default as MPopoverBubble } from './src/PopoverBubble.vue';
```

Comment:
I don't think we should make this public. Rather as an internal only component the public component would still be "Popover" and we'd pass any relevant props from the single-use popover component down into the internal child.

Reply:
I have to disagree here. I don't think we should be too prescriptive on the content of the popover. Exposing this provides a good/useful default, but something like the mini cart will need to have something different (or at least add additional styles).

In my opinion, the popover component should _only_ be responsible for creating the popper instance, and what goes inside that instance should be up to the consumer.

Reply:
While I still don't think the child component is necessary, it is a pattern we established with Modal / Dialog and is reasonable to continue. With the changes for padding, this also makes it easier to override the default padding on that component directly. Making note in this resolution we changed the name to PopoverContainer.